src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/ReportPageTests.kt

```diff
@@ -25,11 +25,39 @@ class ReportPageTests : SeleniumTest()
                 .ensureUserGroupHasPermission("test.user@example.com",
                         ReifiedPermission("reports.read", Scope.Global()))
 
+
         loginWithMontagu()
-        driver.get(RequestHelper.webBaseUrl)
         driver.get(RequestHelper.webBaseUrl + "/reports/testreport/v1/")
 
         assertThat(driver.findElement(By.cssSelector("h1")).text).isEqualTo("display name testreport")
     }
 
+    @Test
+    fun `can publish report`()
```

Comment:
Could do with one more test that non-reviewers don't see the publish switch

Reply:
https://github.com/vimc/orderly-web/pull/45/files#diff-7119cf549a679bbeefd1525c27c5f82eR74